app/assets/stylesheets/timelines.css.sass

```diff
@@ -119,37 +119,11 @@ select
 .tl-toolbar-container *
   vertical-align: bottom
 
-.tl-icon-postponed,
-.tl-icon-preponed,
-.tl-icon-added,
-.tl-icon-deleted,
-.tl-icon-changed,
-.tl-icon-calendar,
-.tl-icon-zoomin,
-.tl-icon-zoomout,
-.tl-icon-outline
-  cursor: default
-  background-position: left center
-  background-repeat: no-repeat
-  padding-left: 19px
-
-.tl-icon-postponed
-  background-image: image-url("bullet_red.png")
-
-.tl-icon-preponed
-  background-image: image-url("bullet_green.png")
-
-.tl-icon-added
-  background-image: image-url("added.png")
-
-.tl-icon-deleted
-  background-image: image-url("webalys/deleted.png")
-
-.tl-icon-changed
-  background-image: image-url("webalys/arrow_left_right.png")
-
-.tl-icon-calendar
-  background-image: image-url("webalys/calendar.png")
+.tl-icon-postponed.icon-time:before
```

Comment:
Didn't you remove all the `tl-icon-*` classes?

I can only find a reference to this specific class only here and in a test...